docs/development/shareable-configs.md

```diff
@@ -55,4 +55,6 @@ An additional benefit of using source code hosting is that the same token/authen
 | GitHub with preset name | `github>abc/foo:xyz` | `xyz`     | `https://github.com/abc/foo`         | `xyz.json`                        |
 | GitLab default          | `gitlab>abc/foo`     | `default` | `https://gitlab.com/abc/foo`         | `default.json` or `renovate.json` |
 | GitLab with preset name | `gitlab>abc/foo:xyz` | `xyz`     | `https://gitlab.com/abc/foo`         | `xyz.json`                        |
+| Gitea default           | `gitea>abc/foo`      | `default` | `https://gitea.com/abc/foo`          | `default.json` or `renovate.json` |
+| Gitea with preset name  | `gitea>abc/foo:xyz`  | `xyz`     | `https://gitea.com/abc/foo`          | `xyz.json`                        |
 | Local default           | `local>abc/foo`      | `default` | `https://github.company.com/abc/foo` | `default.json` or `renovate.json` |
```

Comment:
Not sure why we have duplicated this table ? I think we can remove one of them from `shareable-configs` or `config-presets`

Reply:
Yeah I noticed that as well. I think it's for ease of use for the readers. Because the table is helpful in the `shareable -configs`, and in the `config-presets` documentation.

Duplicating things is never good, so we probably should make a "re-usable component" out of the table, and then insert this component into the docs in the relevant places. This way we have one source of truth for the table.

Reply:
I issue #7924 to discuss this problem.